third_party/libunwindstack/PeCoffInterface.cpp

```diff
@@ -346,18 +346,26 @@ bool PeCoffInterfaceImpl<AddressType>::InitSections() {
     section.vmsize = section_header.vmsize;
     section.offset = section_header.offset;
     section.size = section_header.size;
+    section.flags = section_header.flags;
     sections_.emplace_back(section);
   }
 
   for (size_t i = 0; i < sections_.size(); ++i) {
-    if (sections_[i].name == ".text") {
+    // Find the .text section as the first section with characteristics IMAGE_SCN_CNT_CODE and
+    // IMAGE_SCN_MEM_EXECUTE. We prefer this to looking for a section with name ".text", because we
+    // have observed that this is not very reliable: for example, changing the section names can be
+    // used as a simple means of obfuscation.
+    constexpr uint32_t kImageScnCntCode = 0x00000020;
+    constexpr uint32_t kImageScnMemExecute = 0x20000000;
```

Comment:
I could also declare some of these constants in a more "global" location and use them also in the other places in this change where flags are specified. But there doesn't seem to be a precedent so I'm not sure where. The `PeCoff` or `PeCoffInterface` classes are obvious candidates.

Reply:
I don't feel strongly about either option. You can just leave things as there are. If you put them in header to be used in the tests etc., I think I would prefer PeCoffInterface.h over PeCoff.h. It's a bit lower level and already contains info about the PE/COFF file structure and contents.